lib/datasource/remote/api/proposals_repository.dart

```diff
@@ -39,6 +39,28 @@ class ProposalsRepository extends NetworkRepository with EosRepository {
         .catchError((error) => mapHttpError(error));
   }
 
+  Future<Result<VoteCycleModel>> getCurrentVoteCycle() {
+    print('[http] get vote cycle');
+
+    final request = createRequest(
+      code: accountFunds,
+      scope: accountFunds,
+      table: tableCycleStats,
+      // ignore: avoid_redundant_argument_values
```

Comment:
i would just not pass the argument. This needs to be one. Default is 1. 
If default changes, this probably needs to change too. 

But again, i could go either way in this scenario!.

Reply:
Yeah - it seems to be a bug in the linter... this needs to have 1 passed. Not passing 1, because 1 is the default, runs the danger of missing the check later on when we set the default to 2, there won't be any warnings on this.

Reply:
This linter is detrimental to code quality.